File: palindrome.py
word1=input("Input word to reverse: ")
x=1
word1=word1.lower()
wordreverse=word1[len(word1)-1]

while x <= (len(word1)-1):
	x=x+1
	wordreverse=wordreverse+word1[(len(word1)-x)]
if(word1==wordreverse):
	print("You win it's a palindrome")
else:
	print("It's not a palindrome")
	

	

File: palindrome.rb
puts "This progam checks to see if a word is a palindrome."
puts "Enter your word: "
wordCheck=gets
wordCheck=wordCheck.chomp.downcase
revWord=wordCheck.downcase.reverse
puts wordCheck
puts revWord
if revWord==wordCheck
	puts "You win. It's a palindrome."
else
	puts "It's not a palindrome."
end

	

